README.md

```diff
@@ -137,7 +145,11 @@ provided at the top-level will be the default values for each stream.:
 - `api_url`: required: the base url/endpoint for the desired api.
 - `pagination_request_style`: optional: style for requesting pagination, defaults to `default` which is the `jsonpath_paginator`, see Pagination below.
 - `pagination_response_style`: optional: style of pagination results, defaults to `default` which is the `page` style response, see Pagination below.
-- `use_request_body_not_params`: optional: sends the request parameters in the request body. This is normally not required, a few API's like OpenSearch require this. Defaults to `False`"
+- `use_request_body_not_params`: optional: sends the request parameters in the request body. This is normally not required, a few API's like OpenSearch require this. Defaults to `False`.
+- `backoff_type`: optional: The style of Backoff [message|header] applied to rate limited APIs. Backoff times (seconds) come from response either the `message` or `header`. Defaults to `None`.
+- `backoff_param`: optional: the header parameter to inspect for a backoff time. Defaults to `Retry-After`.
+- `backoff_time_extension`: optional: An addition extension (seconds) to the backoff time over and above a jitter value - use where an API is not precise in it's backoff times. Defaults to `0`.
+- `store_raw_json_message`: optional: An additional extension which will emitted the whole message into an field `_sdc_raw_json`. Useful for a dynamic schema which cannot be automatically discovered. Defaults to `False`.
```

Comment:
Is this the best way to enable this capability? Or would it be better to just toggle the dynamic schema generation on/off?

Reply:
I still need to optionally store the full message in a column in my target e.g. _sdc_raw_json. Where I am doing this, I am also picking a few attributes which have been flattened. In some cases storing the full message along the flattened attributes is desirable. Given this is an optional feature, I think there is no harm.

I have one example with a FHIR API extensions that standard flattening is not desirable as the extension is implemented over two key pairs - the first describing the name of the key and the datatype, the next the value for key. By having the full raw message, there is no data loss if I need to parse it differently.